spot_driver/spot_driver/ros_helpers.py

```diff
@@ -1,10 +1,12 @@
 import os
+import time
```

Comment:
This is my change and should be reverted - sorry!  Mistake while doing integration on Friday.